src/Native/build-native.sh

```diff
@@ -243,18 +248,25 @@ while :; do
         arm64)
             __BuildArch=arm64
             ;;
-        debug)
+        debug|-debug)
             __BuildType=Debug
             ;;
-        release)
+        release|-release)
             __BuildType=Release
             __CMakeArgs=RELEASE
             ;;
+        --osgroup*|-osgroup*)
+            # msbuild adds extra - for -OSGroup
+            __BuildOS=${1#*=}
+            __OSGroupSet=1
+            ;;
         freebsd)
```

Comment:
Since you have added the option with dash at the beginning for `debug` and `release` options, I think we should do that for all the options, like we have it in coreclr.
Could you also please add the `usage()` function to reflect that change (print only the new versions with dash) and to add the osgroup there?

Reply:
I reverted the OSGroup change. Would it make more sense to do help and parameter cleanup as separate PR? That is really not part of the freebsd. I would be happy to take a look at coreclr and corefx and make them as similar as possible as far as build behavior.

Reply:
I have suggested that only because you went half the way and modified only the `debug` and `release` options and left all the rest untouched. I am ok with just reverting that change instead of changing all the options.